tests/nodeUnit.js

```diff
@@ -133,28 +134,40 @@ define(require => {
       });
 
       bdd.it('should cache files', () => {
-        const testURL = 'https://raw.githubusercontent.com/mozilla/platform-status/master/package.json';
+        const testURL = 'http://somejsonurl.com';
+        const testObject = { some: 'value' };
 
-        // Cache our package.json file
-        return cache.readJson(testURL, 5).then(originalText => {
-          // Cause the next fetch to fail
-          fetchMock.mock(testURL, 404);
+        nock(testURL)
+        .get('/')
+        // XXX if eu is caching does it need to call the URL second time?
```

Comment:
Can you try adding the other cache headers (`Expires`, `Last-Modified`, `ETag`, `Pragma`) and see if you still need to call it the second time?

It'd be worth testing:
- resource not expired, eu shouldn't perform a second request;
- expired resource, the server returns 304, so eu uses the cached content;
- expired resource, the server returns 200, so eu uses the new content.